extensions/orshot/src/create-website-screenshot.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  showToast,
+  Toast,
+  getPreferenceValues,
+  LaunchProps,
+} from "@raycast/api";
+import { writeFileSync } from "fs";
+import { join } from "path";
+import { homedir } from "os";
+
+interface Preferences {
+  apiKey?: string;
+}
+
+interface Arguments {
+  websiteUrl: string;
+}
+
+interface ApiResponse {
+  data?: {
+    content?: string;
+  };
+  error?: string;
+}
+
+export default async function Command(
+  props: LaunchProps<{ arguments: Arguments }>,
```

Comment:
```suggestion
  props: LaunchProps<{ arguments: Arguments.CreateWebsiteScreenshot }>,
```
Arguments is also auto-generated.